src/LtiGrade.php

```diff
@@ -58,7 +58,7 @@ public function getScoreGiven()
         return $this->score_given;
     }
 
-    public function setScoreGiven($value)
+    public function setScoreGiven($value): self
```

Comment:
The parameters and class attributes in each function are not typed; is that intentional?

Reply:
It is. I have no idea if people will be passing in `int|float|string|SomeWackyWTF` to these functions. They're just getting used to build a payload to call the LMS, and the IMSGlobal spec doesn't specify types for these. So I'm gonna let go and let god with these types.